markets/perps-market/hardhat.config.ts

```diff
@@ -21,7 +21,7 @@ const config = {
     templates,
   },
   mocha: {
-    timeout: 200000,
+    timeout: 120000,
```

Comment:
lol

Reply:
haha wait, I thought I increased it...Will revert this